src/android/CHIPTool/app/src/main/cpp/native-lib.cpp

```diff
@@ -1,20 +1,4 @@
-/*
- *   Copyright (c) 2020 Project CHIP Authors
- *   All rights reserved.
- *
- *   Licensed under the Apache License, Version 2.0 (the "License");
- *   you may not use this file except in compliance with the License.
- *   You may obtain a copy of the License at
- *
- *       http://www.apache.org/licenses/LICENSE-2.0
- *
- *   Unless required by applicable law or agreed to in writing, software
- *   distributed under the License is distributed on an "AS IS" BASIS,
- *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *   See the License for the specific language governing permissions and
- *   limitations under the License.
- *
- */
+/* See Project CHIP LICENSE file for licensing information. */
```

Comment:
nit: Prefer C++ style comment?

Reply:
I considered this, however our headers use this and quickly followed by a `/* @brief .... */` and I figured mixing of `//` and `/*` would look ugly. Script is not smart enough to detect comment styles here.

We could do a followup pass to use C++.

Reply:
This file in particular would probably be better with // given the style of what follows.
The next one though shows the follow up in `/*`.

The script only handles `/*...*/` actually ... so it replaced C-style with C-style for now.

Reply:
Can we have these link to the actual license?